docusaurus.config.js

```diff
@@ -282,6 +282,6 @@ module.exports = {
       },
     ],
     ['./plugins/dotenv', {}],
-    ['./plugins/hotjar', {}],
+    // ['./plugins/hotjar', {}],
```

Comment:
Why disable the hotjar plugin ?

Reply:
Its a request from @lorenzi-nicolas for SEO

Reply:
Hotjar is a tool that allowed us to release the new version of the landing page and globally to understand the behavior of our visitors on our pages. Hotjar at the SEO level increases significantly the loading time of the pages, so we decompose it to increase the performance of our content platform. It is not excluded that in the future we will use this tool again as part of a new behavioral study.